src/config_creature.c

```diff
@@ -1645,14 +1636,6 @@ TbBool load_creaturetypes_config_file(const char *textname, const char *fname, u
           WARNMSG("Parsing %s file \"%s\" experience block failed.",textname,fname);
     }
     if (result)
-    {
-        result = parse_creaturetype_instance_blocks(buf, len, textname, flags);
```

Comment:
this looks like the reading of the block itself, so if you remove it it won't load the instances at all